packages/dx-react-scheduler/src/plugins/editing-state.tsx

```diff
@@ -13,7 +12,25 @@ import {
   changedAppointmentById,
 } from '@devexpress/dx-scheduler-core';
 
-export class EditingState extends React.PureComponent {
+import { EditingStateProps, EditingStateState } from '../types';
+
+class EditingStateBase extends React.PureComponent<EditingStateProps, EditingStateState> {
+  startEditAppointment: ComputedFn;
+  stopEditAppointment: (payload?: any) => void;
```

Comment:
It is of `ActionFn` type (according to the way it is used in code).

Reply:
I use these functions with and without arguments. There are arguments always exist in the `ActionFn`.

Reply:
The fact that the function is passed to `action` property states that the function **is** of action handler type.

Maybe `ActionFn` signature should be updated to allow optional arguments?

Reply:
These functions are not only `action` functions. Can we keep the current type?

Reply:
If we change the main `AcctionFn` type, we get problems in many other places

Reply:
Let it stay as it is now (until better times).